joplin/css/pages/edit.scss

```diff
@@ -79,6 +79,7 @@
 }
 
 .tab-nav {
+    // this is also giving position fixed to the tab nav in add user
     position: fixed;
```

Comment:
I created an issue highlighting this bug, this note is for future us to find where to fix it